src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -743,9 +743,13 @@ ProActive User Space mapped using SFTP has the following benefits:
 It has the following constraints and limitations:
 
  * Every user's credentials of the ProActive server must match system users credentials inside the SFTP server. This is typically achieved by enabling LDAP or PAM authentication on the ProActive server (see <<_user_authentication>>).
- * Every file transfer will use the SFTP protocol when it's configured. Note that SFTP protocol will be less efficient than the ProActive protocol when transferring files on the local file system.
+ * Every file transfer will use the SFTP protocol when it's configured. Note that SFTP protocol will be less efficient than the ProActive protocol when transferring files onto and from the local file system.
 
-The SFTP protocol can also be used for the Global Space, but in that case, all file transfers will be accomplished by a single system account.
+The SFTP protocol can also be used for the Global Space.In that case, all file transfers will be accomplished by a single system account.
```

Comment:
`space` In that case, all file transfers **are** accomplished by a single system account.

Reply:
Add space before In
Space.In that
-> 
Space. In that

Reply:
yes